unity-renderer/Assets/Tutorial/Scripts/TutorialConfigurator.cs

```diff
@@ -0,0 +1,48 @@
+using Cinemachine;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.Tutorial
+{
+    public class TutorialConfigurator : MonoBehaviour
```

Comment:
Not sure if you already did it, but we can use this opportunity to remove the `TutorialController` from `InitialScene`.

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/966/commits/64e96180bad0746ca4f7146eada3252b14aed9b4. Thanks for the feedback!